static/library/ranking_component.ts

```diff
@@ -61,26 +65,64 @@ export class DatacommonsRankingComponent extends LitElement {
   @property()
   apiRoot: string;
 
+  // Type of child place to rank (ex: State, County)
+  @property()
+  childPlaceType!: string;
+
+  @property()
+  diffBaseDate?: string;
```

Comment:
add a comment for what this is? But also, can you check if this is even used anywhere & maybe we can get ride of it

Reply:
Checked, this doesn't appear to be used anywhere. Removed all references.